x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/meta_engines_table_expanded_row.scss

```diff
@@ -0,0 +1,22 @@
+
+.meta-engines__source-engines-table {
+  margin: -8px -8px 7px -8px;
+
+  thead {
+    display: none;
+  }
+
+  @include euiBreakpoint('l', 'xl') {
+    .euiTableRowCell {
+      border-top: none;
+    }
+
+    .euiTitle {
+      display: none;
+    }
+  }
```

Comment:
Is there a better way to accomplish this?

Reply:
After thinking about this and how the expanded row works I don't think so, currently. One possible option would be to not nest an EuiBasicTable within the expanded row and instead try to directly use cell components, e.g. `<EuiTableRowCell>` - but then you lose some of the automatic mobile UX that comes with the basic table.

I think this is fine for now, especially since after testing the expanded row behavior in a screenreader, it appears that EUI's implementation of expanded rows has a known issue where screen readers cannot dive into the expanded row, so it's not really possible to test this accessibility-wise anyway.

It might be interesting to revisit it someday to try and compare implementations, but I'm definitely good with a nested table for now (which I learned today is semantically valid! 🤯)